### DISCUSSION TITLE
Upgrading Unreal Engine Version from 4.24 to 4.25

### DISCUSSION THREAD

**Discussion by hoangvietdo** [microsoft/AirSim#4319] [General]
Does anyone try to run a lower UE project under higher UE repo? Specifically, I have an 4.24 UE project that I created from Window, I copied the project folder to Ubuntu 18.04 machine under 4.25 UE and run the project. There was a error message about compatible and I selected rebuild.  After loading, my project environment become empty (i.e., only 2 chairs and 1 table). In addition, I also tried to upgrade my project from 4.24 to 4.25 using a Window machine and after upgrading, my project also becomes empty.

P/s:  When I was creating the environment, I just simply dragged the item from some assets and run, so I assumed that my UE project is a blueprint one.

**Comment by zimmy87** [comment]
I personally haven't run into any issue with UE's "switch version..." context menu option in windows when upgrading (downgrading, on the other hand, has never worked for me), though I haven't personally tried going from 4.24 to 4.25. This sounds like an issue with UE's upgrade functionality, so I think https://answers.unrealengine.com/index.html may be a better place for this question as you should get more visibility from UE experts there. If you can provide log output or upload your 4.24 project, it should help them debug further.
